src/protocol/tx-validity.md

```diff
@@ -174,9 +174,12 @@ def metadata_gas_fees(tx) -> int:
                 # add intrinsic cost of calculating the contract id 
                 # size = 4 byte seed + 32 byte salt + 32 byte code root + 32 byte state root
                 total += sha256_gas_fee(100)
-                # add intrinsic cost of calculating the transaction id
-                total += sha256_gas_fee(size(tx))
-    elif tx.type == TransactionType.Script:
```

Comment:
Wait, what happened to the `Script` path?

Reply:
It is handled below=) 
<img width="534" alt="image" src="https://github.com/FuelLabs/fuel-specs/assets/18346821/1f5f41a0-5eb5-4607-9f8f-39c51e054fe0">

Since that part is identical for all transactions except `Mint`